front/lib/actions/mcp_internal_actions/servers/tables_query/server_v2.ts

```diff
@@ -177,9 +183,15 @@ function createServer(
         }
         const tableConfigurations = tableConfigurationsRes.value;
         if (tableConfigurations.length === 0) {
-          return makeMCPToolTextError(
-            "The agent does not have access to any tables. Please edit the agent's Query Tables tool to add tables, or remove the tool."
-          );
+          return {
+            isError: false,
```

Comment:
I still believe this should be an isError: true no? The tool errors with a validation error here and the model should see that it's an error and not a success right?